libs/langgraph/tests/test_pregel.py

```diff
@@ -7632,10 +7663,6 @@ def decider(data: State) -> str:
 
     app = workflow.compile()
 
-    assert app.get_graph().draw_mermaid(with_styles=False) == snapshot
```

Comment:
why is this being removed?

Reply:
can add it back, originally removed because it was a pydantic v1 test, but we can keep it